src/consts.h

```diff
@@ -30,11 +30,15 @@ typedef enum {
     TS_AGG_MAX,
     TS_AGG_SUM,
     TS_AGG_AVG,
+    TS_AGG_STD_P,
+    TS_AGG_STD_S,
+    TS_AGG_VAR_P,
+    TS_AGG_VAR_S,
     TS_AGG_COUNT,
     TS_AGG_FIRST,
     TS_AGG_LAST,
     TS_AGG_RANGE,
-    TS_AGG_TYPES_MAX // 9
+    TS_AGG_TYPES_MAX // 10
```

Comment:
definitely not 10

Reply:
You are right. I missed this when expanded with more aggs.

Reply:
was corrected